includes/admin/assets.php

```diff
@@ -136,10 +136,18 @@ public function load()
 				wp_enqueue_style('simcal-admin-add-calendar');
 			}
 		}
-		// Load the style in admin setting page only.
+		wp_enqueue_style('sc-tail-style', $css_path . 'tailwind-output-style.css', [], SIMPLE_CALENDAR_VERSION);
```

Comment:
This will load the CSS is all admin pages which is wrong

Reply:
can we add this in all condition? or can we make the separate condition for that.

Reply:
@sujitxtendify Yes please add condition to check is the post type is simple calendar like (post_type=calendar)

Reply:
i had added condition already.